Expensify/App
# #3293 update last message details on message deletion

### Details

Fixes incorrect display of last message on LHN after deleting last message of the chat.

### Fixed Issues

Fixes #3293

### Tests/QA Steps



1. Log in to the app
2. Navigate to a conversation
3. Send a message
4. Delete the message
5. Message in LHN should be updated in a few moments.
6. Refresh the page
7. Message shown on LHN should be the last one you have in that chat.

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/64093836/121597359-4ab91900-ca49-11eb-94ab-b222797a391e.mp4

#### Mobile Web


https://user-images.githubusercontent.com/64093836/121597374-50166380-ca49-11eb-9ed9-216fc2be52e0.mp4

#### Desktop


https://user-images.githubusercontent.com/64093836/121598942-0d558b00-ca4b-11eb-81a3-499fb456a62b.mov



#### iOS


https://user-images.githubusercontent.com/64093836/121599027-2c541d00-ca4b-11eb-84f4-cead49f188b6.mp4


#### Android


https://user-images.githubusercontent.com/64093836/121598113-21e55380-ca4a-11eb-9d2e-1b385614db41.mp4
